### DISCUSSION TITLE
Rebuilds during updates?

### DISCUSSION THREAD

**Discussion by adrian5** [nvim-pack/nvim-spectre#187] [Q&A] [answered]
Lately I've had 3–4 rebuilds (rust) when nvim-spectre got an update. Is this actually necessary or a side effect I can suppress via the plugin config? I find it rather disruptive compared to other (lua) plugin updates and don't use spectre that often.

**Comment by EpsilonKu** [comment, marked as answer]
I guess it's relevant to #186. Making MR

**Comment by EpsilonKu** [comment]
Sorry I mean #185 . I already made MR. Can you check it?

**Reply by adrian5** [reply]
Would this take effect despite the latest PR being the assumed fix? Wouldn't I need to wait for another update _after_ that one to see if it changed?

**Reply by EpsilonKu** [reply]
Yeah, I recommend to wait.

**Reply by adrian5** [reply]
I did try myself with a fork, but all my experiments failed. I also tried working with a local copy using lazy.nvim's `dir` option to iterate faster, but I don't think I can trigger "update" behavior that way.
